packages/kernel-6.12/config-bottlerocket

```diff
@@ -0,0 +1,255 @@
+# Because Bottlerocket does not have an initramfs, modules required to mount
+# the root filesystem must be set to y.
+
+# The root filesystem is ext4 or erofs
+CONFIG_EXT4_FS=y
+CONFIG_EROFS_FS=y
+
+# With EROFS, we do not use SQUASHFS
+# CONFIG_SQUASHFS is not set
+
+# ARM: EFI_ZBOOT
+CONFIG_EFI_ZBOOT=y
+
+# NVMe support
+CONFIG_BLK_DEV_NVME=y
+CONFIG_NVME_CORE=y
+
+# kernel-hardening-checker recommendations
+# CONFIG_ACPI_TABLE_UPGRADE is not set
+CONFIG_ARCH_MMAP_RND_BITS=33
+CONFIG_ARCH_MMAP_RND_COMPAT_BITS=16
+# CONFIG_BLK_DEV_WRITE_MOUNTED is not set
+# CONFIG_CACHESTAT_SYSCALL is not set
+CONFIG_DEBUG_NOTIFIERS=y
+CONFIG_DEBUG_SG=y
+# CONFIG_EFI_CUSTOM_SSDT_OVERLAYS is not set
+CONFIG_EFI_DISABLE_PCI_DMA=y
+# CONFIG_HIBERNATION is not set
+CONFIG_INIT_ON_ALLOC_DEFAULT_ON=y
+CONFIG_INTEL_IOMMU_DEFAULT_ON=y
+CONFIG_INTEL_IOMMU_SVM=y
+CONFIG_IOMMU_DEFAULT_DMA_STRICT=y
+CONFIG_KFENCE_SAMPLE_INTERVAL=100
+CONFIG_KFENCE=y
+# CONFIG_LEGACY_TIOCSTI is not set
+# CONFIG_LIVEPATCH is not set
+# CONFIG_MAGIC_SYSRQ is not set
+CONFIG_MAGIC_SYSRQ_DEFAULT_ENABLE=176
+CONFIG_RANDOM_KMALLOC_CACHES=y
+CONFIG_RANDOMIZE_BASE=y
+CONFIG_RANDOMIZE_KSTACK_OFFSET_DEFAULT=y
+CONFIG_RESET_ATTACK_MITIGATION=y
+CONFIG_X86_USER_SHADOW_STACK=y
```

Comment:
I'd like to pull these out and have them come in a follow-up PR, so it's easier to do bisect builds before and after the changes to check for performance impacts.